packages/opentelemetry-sdk-trace-base/src/Tracer.ts

```diff
@@ -120,15 +124,23 @@ export class Tracer implements api.Tracer {
       return api.trace.wrapSpanContext(spanContext);
     }
 
+    let clock: AnchoredClock | undefined;
+    if (parentSpan) {
+      clock = this.clocks.get(parentSpan);
```

Comment:
What is reason for reusing an `AnchoredClock` from parentSpan?
This fails for sure for remote parents and non sampled parents.
if there is a long running parent the anchor would be (mostly) sticky for the part of the trace in this process. Not sure if this is a good or a bad thing.

Reply:
We can not assert the monotonicity between different AnchoredClock instances as it uses the `Date.now` as its anchor. To prevent creating a child span that has a start time earlier than its parent span, I believe we need to share their anchored clock.

Reply:
Yes exactly. There should be a single anchored clock per local trace. If the system clock changes during a trace, we want to be sure we're using the same anchor for that whole trace.